api/views/phones.py

```diff
@@ -464,16 +467,21 @@ def inbound_call(request):
         raise exceptions.ValidationError("Call data missing Caller or Called.")
 
     relay_number, real_phone = _get_phone_objects(inbound_to)
+
     _check_disabled(relay_number, "calls")
+    _check_remaining(relay_number, "minutes")
     inbound_contact = _get_inbound_contact(relay_number, inbound_from)
     if inbound_contact:
-        _check_and_update_contact(inbound_contact, "calls")
+        _check_and_update_contact(inbound_contact, "calls", relay_number)
 
     # Note: TwilioInboundCallXMLRenderer will render this as TwiML
+    relay_number.calls_forwarded += 1
+    relay_number.save()
```

Comment:
Nit: move these lines above the line `# Note: TwilioInboundCallXMLRenderer...`, so that comment remains close to `return response.Response`